### PR TITLE
Docs review

### PR DESCRIPTION
**Pull request by ravescovi**
Document and deployment review.

### CODE REVIEW

File: gladier_xpcs/flows/flow_eigen.py [hunk 1]
```diff
@@ -30,11 +37,11 @@ class XPCSEigen(ContainerBaseClient):
...
-        'gladier_xpcs.tools.MakeCorrPlots',
-        'gladier_xpcs.tools.gather_xpcs_metadata.GatherXPCSMetadata',
-        'gladier_xpcs.tools.Publish',
+        TransferFromClutchToTheta,
+        PrePublish,
+        AcquireNodes,
+        EigenCorr,
+        MakeCorrPlots,
+        GatherXPCSMetadata,
+        Publish,
```
Comment (NickolausDS) [inline gladier_xpcs/flows/flow_eigen.py:46]: Looks like the changes from #110 made it in here, you can remove them with a `git rebase -i head^^^`

**Comment by NickolausDS** [comment]
The docs changes are a great start! I realize the names changed from the last refactor, so it's good that the new ones carry over. I wonder if we should add a note to 'reprocessing' to mention it isn't "production ready" yet?

**Comment by NickolausDS** [comment]
The docs changes are a great start! I realize the names changed from the last refactor, so it's good that the new ones carry over. I wonder if we should add a note to 'reprocessing' to mention it isn't "production ready" yet?

**Comment by ravescovi** [comment]
1 - Is reprocessing a different flow now? If not, can we phase out the old
one? I will add the note on the production side.
2 - Does it make sense to add a glossary and figures?

**Comment by NickolausDS** [comment]
I glossed over the code, and I still see references to the same reprocessing flow. I think we'll still need to keep it as a separate flow in order to publish it as a separate dataset and to not overwrite the original, so it's probably best to leave it alone for now until we can carve out time to work on it again. 

For adding a glossary and figures, I don't think we really need them but if you had them handy it might make for nice additions. Probably a bit overkill IMO.

File: README.md [hunk 2]
```diff
@@ -1,38 +1,37 @@
...
 
+---
+## XPCS Reprocessing 
 
-## Reprocessing
-
-XPCS Reprocessing takes data already published in the portal and re-runs it on corr with
-a customized (with a qmap file) hdf file. Reprocessing also has an extra step to rename
-the dataset to publish it under a different title in the portal. 
+XPCS Reprocessing takes data already published in the portal and re-runs the main flow (above)  with a customized (with a qmap file) hdf file. This flow has an extra step to rename the dataset to publish it under a different title in the portal.
```
Comment (NickolausDS) [inline README.md:34]: Maybe add a note below this to tell people this is not a 'production' feature?

File: README.md [hunk 3]
```diff
@@ -79,6 +94,14 @@ Please add, update, or correct information as things change.
...
 ```
 
+### Beamline Setup
+
+
+
+### Remote Setup
+
+
+
```
Comment (NickolausDS) [inline README.md:104]: What are these two sections for?

File: gladier_xpcs/flows/flow_eigen.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (NickolausDS) [reply gladier_xpcs/flows/flow_eigen.py:46]: Looks like these changes are still here?

**Pull request closed without merging**